docs/USER-GUIDE.md

```diff
@@ -256,21 +256,17 @@ This simply stores the data as a compressed tarball in a Google Cloud Storage bu
 
 ```yaml
 config:
-  # Defect dojo configuration
   googleCloudStorage:
     keyFile: "/path/to/GCS/key"                           # optional: path to the GCS key file (alternatively: use GOOGLE_APPLICATION_CREDENTIALS)
-    bucketName: "<name-of-GCS-bucket-to-export-to>"       # Mandatory
-    directory: "<override-of-default-directory>"          # Optional directory where the credentials have write access, defaults to `RapiDAST-<product>`
+    bucketName: "<name-of-GCS-bucket-to-export-to>"       # Required
+    directory: "<name-of-directory-under-bucket>"         # Required, GCS directory(folder) name where write access is granted
 ```
 
-Once this is set, scan results will be exported to the bucket automatically. The tarball file will include:
-
- 1. metadata.json - the file that contains scan_type, uuid and import_data(could be changed later. Currently this comes from the previous DefectDojo integration feature)
```

Comment:
Since the metadata.json file with the`scan_type` field is no longer present, I suppose consumers of this data can infer the scan type based on string comparison with directory names? E.g. directory begins with "zap*" so scan is zap type?

Reply:
Yes. zap results are found under the "zap" directory and garak results are found under the "garak" directory. The directory name is created from the scanner name.